nima/tests/integration/webserver/access-log/src/test/java/io/helidon/nima/tests/integration/server/accesslog/AccessLogTest.java

```diff
@@ -86,10 +86,17 @@ void testRequestsAndValidateAccessLog() {
                 .request();
         assertThat(response.status(), is(Http.Status.BAD_REQUEST_400));
 
-        assertThat(LOG_HANDLER.get().contains(
-                "127.0.0.1 - [03/Dec/2007:10:15:30 +0000] \"GET /access HTTP/1.1\" 200",
-                "127.0.0.1 - [03/Dec/2007:10:15:30 +0000] \"GET /wrong HTTP/1.1\" 404"),
-                is(true));
+        try {
+            assertThat(LOG_HANDLER.get().contains(
+                            "127.0.0.1 - [03/Dec/2007:10:15:30 +0000] \"GET /access HTTP/1.1\" 200",
+                            "127.0.0.1 - [03/Dec/2007:10:15:30 +0000] \"GET /wrong HTTP/1.1\" 404"),
+                    is(true));
+        } catch (AssertionError e) {
+            System.out.println("-----");
```

Comment:
`System.err.println`, surely?

Reply:
Just a test as it is only failing in the pipeline. In any case, that code was updated already.